internal/hidrpc/message.go

```diff
@@ -11,9 +31,22 @@ type Message struct {
 	d []byte
 }
 
+func (m *Message) MarshalZerologObject(e *zerolog.Event) {
```

Comment:
Allow zerolog to serialize the object cleanly and fast without reflection if you use `.Object()`